openadapt/app/dashboard/app/recordings/RegularRecordings.tsx

```diff
@@ -0,0 +1,44 @@
+import { SimpleTable } from '@/components/SimpleTable';
+import { Recording } from '@/types/recording';
+import React, { useEffect, useState } from 'react'
+import { timeStampToDateString } from '../utils';
+import { useRouter } from 'next/navigation';
+
+export const RegularRecordings = () => {
```

Comment:
`Regular` is intended to differenate vs. `Scrubbed`, right? If so, what do you think about renaming this to `RawRecordings`, or just `Recordings`?